packages/shared-metrics/src/util/nativeModuleRetry.js

```diff
@@ -184,22 +184,9 @@ function copyPrecompiled(opts, loaderEmitter, callback) {
         // See below for the reason why we append 'precompiled' to the path.
         const targetDir = path.join(opts.nativeModulePath, 'precompiled');
 
-        // @ts-ignore
-        copy(
-          path.join(os.tmpdir(), opts.nativeModuleName),
-          targetDir,
-          {
-            overwrite: true,
-            dot: true
-          },
-          // @ts-ignore
-          cpErr => {
-            if (cpErr) {
-              logger.warn(`Copying the precompiled build for ${opts.nativeModuleName} ${label} failed.`, cpErr);
-              callback(false);
-              return;
-            }
-
+        fse
```

Comment:
In the replace-copy method, we pass the options `overwrite: true` (to allow overwriting of destination files) and `dot: true` (to enable copying of files that begin with a dot). Similarly, in `fs-extra`, the default behavior sets `overwrite` to `true` and also supports copying files that begin with a dot(added a test to verify this functionality). For reference, see the documentation at [fs-extra copy method](https://github.com/jprichardson/node-fs-extra/blob/master/docs/copy.md).